Initializes the September newsletter (#201)
readthedocs/ethicalads.io: content/posts/newsletter-september-2021.md

Title: EthicalAds Newsletter - September 2021
Date: October 4, 2021
description: Updates from September, including stats and commentary on our progress as we build EthicalAds
tags: newsletter, community
authors: Ra Cohen
image: /images/posts/2021-september-newsletter.jpg
image_credit: <span>Photo by <a href="https://unsplash.com/@souvenirpixels?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">James Wheeler</a> on <a href="https://unsplash.com/s/photos/pumpkin?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a></span>

## New features

These are the major features we released in the last month:

* We added a read replica database which improves the performance of our dashboard.

> * In our server, when you copy an advertisement to a new flight, they're ordered by the age of the ad, newest first. 

Reviewer: ```suggestion
* In the advertiser dashboard, when you copy an advertisement to a new flight, they're ordered by the age of the ad, newest first. 
```

Reviewer 2: I actually wonder whether this is even worth mentioning. It's a small improvement.

* Now when a 'Contact Us' form is filled out, we ask if they would like to start receiving our newsletter. 

> * Our new [prospectus](https://www.ethicalads.io/prospectus/ethicalads-advertiser-prospectus.pdf) with our updated pricing came out. 

Reviewer: This should go at the top. Perhaps even outside of a bullet point.

Reviewer 2: I would give a small bit on the changes. Specifically, we raised geo-only prices in the EU and North America.

You can always see our latest server updates in our [ethical-ad-server changelog](https://ethical-ad-server.readthedocs.io/en/latest/developer/changelog.html) and [ethical-ad-client changelog](https://ethical-ad-client.readthedocs.io/en/latest/changelog.html).

## September advertising stats

In September 2021:

* We generated **$31,380.78** for our publishers.
* We had **103 publishers** on our network with at least one paid ad impression.

> * We had **19,238,786** paid ad views across the world.

Reviewer: We talked about breaking these numbers down by region, which could be interesting (a small table). Perhaps start in Q4 (next month), if we think that would be interesting/useful.

Author: Sounds like a plan!

September was our best month to date in terms of paid views and revenue! 

> We had a 12% increase in September's revenue from August which was a 15% increase from July before that. 

Reviewer: 🎉

## Tricky bits

> We still have the fortunate problem of being overbooked in the US and EU especially,
> which has resulted in many of our on-going campaigns falling behind of schedule.
> Thus we're adjusting our roadmap priorities to focus more heavily on publisher acquisition.
> 

Reviewer: ```suggestion
We still have the fortunate problem of being overbooked in the US and EU especially,
which has resulted in many of our on-going campaigns falling behind schedule. We've adjusted our roadmap priorities to focus more heavily on publisher acquisition.
We are also looking into additional ways of doing pricing,
so that we are able to better determine priority of ads across time. 

```

Reviewer 2: I would highlight here some of the pricing changes to attract additional publishers. Specifically, I'd show a little data about our NA and EU eCPMs going up and how we're becoming more competitive with the larger display networks that track folks.

## Upcoming features

The major features in our upcoming roadmap:

* New landing pages describing our open source ad network and the benefits of being a data science publisher on our network. 
* Adding graphs to our advertising dashboard.

> * Adding "view time" statistics to our reporting, so that we can better understand how ads are performing.

Reviewer: ```suggestion
```

This has been deprioritized for now.

  We raised the geo-prices in the US and EU in order to combat our overbooking issue and earn more money for our publishers.  
* We added a read replica database which improves the performance of our dashboard.
* In the advertiser dashboard, when you copy an advertisement to a new flight, they're ordered by the age of the ad, newest first. 
* Now when a 'Contact Us' form is filled out, we ask if they would like to start receiving our newsletter. 

You can always see our latest server updates in our [ethical-ad-server changelog](https://ethical-ad-server.readthedocs.io/en/latest/developer/changelog.html) and [ethical-ad-client changelog](https://ethical-ad-client.readthedocs.io/en/latest/changelog.html).

## September advertising stats

In September 2021:

* We generated **$31,380.78** for our publishers.
* We had **103 publishers** on our network with at least one paid ad impression.
* We had **19,238,786** paid ad views across the world.

September was our best month to date in terms of paid views and revenue! 
We had a 12% increase in September's revenue from August which was a 15% increase from July before that. 

> 

Reviewer 2: I'd really like to see a quick month over month eCPM comparison for a few of our top regions here. I'd word it like:

```suggestion

Part of our revenue improvements were due to price increases that went into effect over the last quarter
and partially due to selling more of our harder to sell inventory.
Our effective cost per thousand impressions (eCPM) across the US & Canada has improved from $XX in June to $YY in September
and our eCPM in Asia-Pacific improved from $AA in June to $BB.
This means more money in the pockets of our publishers.
```

## Tricky bits

We still have the fortunate problem of being overbooked in the US and EU especially,

> which has resulted in many of our on-going campaigns falling behind schedule. We've adjusted our roadmap priorities to focus more heavily on publisher acquisition.

Reviewer 2: ```suggestion
which has resulted in many of our on-going campaigns falling behind schedule.
We've adjusted our roadmap priorities to focus more heavily on publisher acquisition.
```

Reviewer 2: This just breaks up a long line.

> We are also looking into additional ways of doing pricing,

Reviewer 2: ```suggestion
We are also looking into additional ways of managing pricing,
```

so that we are able to better determine priority of ads across time. 

## Upcoming features

The major features in our upcoming roadmap:

* New landing pages describing our open source ad network and the benefits of being a data science publisher on our network. 
* Adding graphs to our advertising dashboard.
* Highlighting our 10% discount on ad buys over $3000 in the flight report page.

> 

Reviewer 2: ```suggestion
* Infrastructure improvements to better manage our load and isolate reporting from our live transactional data.
```

Title: EthicalAds Newsletter - September 2021

> Date: October 4, 2021

Reviewer 2: ```suggestion
Date: October 5, 2021
```

* We added a read replica database which improves the performance of our dashboard.
* In the advertiser dashboard, when you copy an advertisement to a new flight, they're ordered by the age of the ad, newest first. 
* Now when a 'Contact Us' form is filled out, we ask if they would like to start receiving our newsletter. 

You can always see our latest server updates in our [ethical-ad-server changelog](https://ethical-ad-server.readthedocs.io/en/latest/developer/changelog.html) and [ethical-ad-client changelog](https://ethical-ad-client.readthedocs.io/en/latest/changelog.html).

## September advertising stats

In September 2021:

* We generated **$31,380.78** for our publishers.
* We had **103 publishers** on our network with at least one paid ad impression.
* We had **19,238,786** paid ad views across the world.

September was our best month to date in terms of paid views and revenue! 
We had a 12% increase in September's revenue from August which was a 15% increase from July before that. 

Part of our revenue improvements were due to price increases that went into effect over the last quarter
and partially due to selling more of our harder to sell inventory.

> Our effective cost per thousand impressions (eCPM) across the US & Canada has improved from $2.97 in June to $YY in September

Reviewer 2: As soon as you have these, this is good to go!
